packages/manager/modules/anthos/src/dashboard/ips/remove-private-ip/remove-private-ip.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('anthos.dashboard.ips.remove-private-ip', {
+    url: '/remove-private-ip',
+    params: { privateIp: null },
+    views: {
+      modal: {
+        component: 'anthosDashboardIpsRemovePrivateIp',
+      },
+    },
+    layout: 'modal',
+    onEnter: /* @ngInject */ ($transition$, goBack) => {
```

Comment:
Why not prevent user from entering the state itself by using `redirectTo` ?

Reply:
I have to test if required object is exist or not